CMakeLists.txt

```diff
@@ -675,6 +675,7 @@ if(VELOX_ENABLE_ARROW)
 endif()
 
 if(VELOX_ENABLE_GEO)
+  add_definitions(-DVELOX_ENABLE_GEO)
```

Comment:
Please use target/file based version and move it into this [if](https://github.com/facebookincubator/velox/pull/13851/files#diff-dd02356b3e62048cff4aea44c6b7202764835996dd5dfa16666cca020a9505afR42)
You can see how it was done for faiss just below.